pintc/tests/basic_tests/aliases.pnt

```diff
@@ -0,0 +1,70 @@
+type MyInt = int;
+type MyReal = real;
+type MyBool = bool;
+type MyTuple = { int, int };
+type MyArray = bool[5];
+type MyMap = (b256 => int);
```

Comment:
I'd also like to see a map type here that uses two aliases as in:
```
type MyMap = ( MyB256 => MyTuple )
```

Also, maybe some quite complex such as 
```
type MyOtherMap = ( MyArray => MyMap )
```

Reply:
`MyOtherMap` isn't allowed right now. The keys for a map must be int, bool, or b256.

```pnt
type MyOtherMap = ( MyArray => MyMap );

results in

Error: compiler internal error: currently in storage maps, keys must be int, bool, or b256 and values must be int or bool
    ╭─[test.pnt:23:5]
    │
 23 │     z: MyComplexMap,
    │     ───────┬───────  
    │            ╰───────── currently in storage maps, keys must be int, bool, or b256 and values must be int or bool
```